tests/foreman/api/test_registration.py

```diff
@@ -227,3 +227,26 @@ def test_negative_global_registration_without_ak(module_target_sat):
     with pytest.raises(HTTPError) as context:
         module_target_sat.api.RegistrationCommand().create()
     assert 'Missing activation key!' in context.value.response.text
+
+
+def test_positive_capsule_without_registration_module(module_target_sat, module_capsule_configured):
+    """Test a capsule without registration module
```

Comment:
With this description it looks more like a Capsule/Installer test, not registration test, so could you add some more description here?

Reply:
I feel the description is valid for the test scenario.

Reply:
some thing like `Verify registration with Capsule, when registration isn't configured in installer`